### PR TITLE
[karavi-observability] Oservability 1.6 updates

### PR DESCRIPTION
**Pull request by donatwork**
# Description
Added/updated samples for Observability 1.6 CSM 2.8 changes.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/885|

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] Unit tests passed.
- [ ] Test B

### CODE REVIEW

File: pkg/modules/observability_test.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func TestObservabilityPrecheck(t *testing.T) {
 
 			tmpCR := customResource
 			observability := tmpCR.Spec.Modules[0]
-			observability.ConfigVersion = "v1.4.0"
+			observability.ConfigVersion = "v1.5.0"
```
Comment (santhoshatdell) [inline pkg/modules/observability_test.go:108]: 1.6.0

**Review by santhoshatdell — APPROVED** [review]

**Comment by santhoshatdell** [comment]
Could you check if bundle/manifests/dell-csm-operator.clusterserviceversion.yaml file needs changes?

File: pkg/modules/observability_test.go (hunk 1 above)
Comment (donatwork) [reply pkg/modules/observability_test.go]: Fixed.

**Review by santhoshatdell — DISMISSED** [review]

**Review by jooseppi-luna — COMMENTED** [review]
Have you tested these by installing observability with these changes or by running operator e2e tests for observability install against the changes?

File: operatorconfig/moduleconfig/common/version-values.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ powerscale:
   v2.8.0:
       authorization: "v1.7.0"
       replication: "v1.5.0"
-      observability: "v1.5.0"
+      observability: "v1.6.0"
       resiliency: "v1.7.0"
 powerflex:
   # List of Driver versions and modules that supports the version
```
Comment (rensyct) [inline operatorconfig/moduleconfig/common/version-values.yaml:19]: Not in the scope of this PR but please help remove entries related to 2.5.0 for powerflex in this file

**Comment by donatwork** [comment]
> Have you tested these by installing observability with these changes or by running operator e2e tests for observability install against the changes?

I used a custom build of my changes and was able to verify that install worked for PowerScale. Checked that all pods were deployed (nightly images).

**Review by santhoshatdell — DISMISSED** [review]

File: operatorconfig/moduleconfig/common/version-values.yaml (hunk 2 above)
Comment (donatwork) [reply operatorconfig/moduleconfig/common/version-values.yaml:19]: Removed.

**Comment by donatwork** [comment]
> Could you check if bundle/manifests/dell-csm-operator.clusterserviceversion.yaml file needs changes?

Made updates to that file.

**Comment by jooseppi-luna** [comment]
> > Have you tested these by installing observability with these changes or by running operator e2e tests for observability install against the changes?
> 
> I used a custom build of my changes and was able to verify that install worked for PowerScale. Checked that all pods were deployed (nightly images).

Awesome, if you could do a similar check for the install with PowerFlex I can approve.

**Comment by donatwork** [comment]
> > > Have you tested these by installing observability with these changes or by running operator e2e tests for observability install against the changes?
> > 
> > 
> > I used a custom build of my changes and was able to verify that install worked for PowerScale. Checked that all pods were deployed (nightly images).
> 
> Awesome, if you could do a similar check for the install with PowerFlex I can approve.

@jooseppi-luna Verified PowerFlex install of driver and Observability (nightly) with changes.

**Review by jooseppi-luna — APPROVED** [review]
awesome, thanks for that -- lgtm

**Pull request merged**